ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -60,6 +60,9 @@ chain:
     - name: HYPERSHIFT_CREATE_CLUSTER_RENDER
       default: "false"
       documentation: "If true, render artifacts to ${SHARED_DIR}/hypershift_create_cluster_render.yaml"
+    - name: HYPERSHIFT_CREATE_CLUSTER_RENDER_SENSITIVE
+      default: "false"
+      documentation: "When used along --render it enables rendering of secrets."
```

Comment:
Please remove this HYPERSHIFT_CREATE_CLUSTER_RENDER_SENSITIVE since you have removed your related code